codebuild/spec/sidetrail/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+
+RUN apt-get -y install software-properties-common
+RUN apt-add-repository ppa:brightbox/ruby-ng
+RUN apt-get update
+RUN apt-get -y install libssl-dev python-pip ruby2.3
```

Comment:
Generally, you want to string as many RUN commands together, where it makes sense, to reduce the number of layers created, e.g. Smack's [Dockerfile](https://github.com/smackers/smack/blob/master/Dockerfile#L15)